ckanext/data_qld/plugin.py

```diff
@@ -1,268 +0,0 @@
-# encoding: utf-8
```

Comment:
Was this functionality moved somewhere else? why is this being purged i thought this was active and needed for data.qld.gov.au to function?

Reply:
As per the comment on https://github.com/qld-gov-au/ckanext-data-qld/pull/96/commits/1c7e021647ffb3a2123339ce70963c7d76b4438e this was split into separate plugins (one shared with Publications, the other not) a while ago.